libs/images/node-base/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:12-buster
+
+ARG NODE_VERSION=12.x
+ARG TYPESCRIPT_VERSION=3.7.5
+
+LABEL maintainer="Amido Stacks <stacks@amido.com>"
+
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
+    apt-get install -y curl grep sed unzip && \
+    apt-get clean
+
+RUN npm install -g typescript@${TYPESCRIPT_VERSION} @types/node
```

Comment:
Does it need tto be global?

Reply:
yeah otherwise we would be assuming the user's app location and when they install with npm ci the node_modules folder will be deleted anyway